parsec/core/fs/exceptions.py

```diff
@@ -49,9 +51,9 @@ class FSOperationError(OSError, FSError):
     Base class for the exceptions that may be raised during the execution of an operation
     """
 
-    ERRNO = None
+    ERRNO: Optional[int] = None
     WINERROR = None
```

Comment:
No typing for WINERROR ?

Reply:
WINERROR never happens, it's stayed None for ever  ! (fixed)